scripts/check-submodule-version.ts

```diff
@@ -0,0 +1,17 @@
+import { exec } from "child_process";
+
+exec('yarn list --pattern typescript', (err, stdout, stderr) => {
```

Comment:
Any reason this can't be one of our `mocha` tests?

Reply:
Assuming the qunit tests are never run in the browser and only in node, then should be fine to use qunit. Is that preferable?

Reply:
Yeah, I think it makes it a bit clearer what we have going on...

Reply:
@rwjblue cool, I've mode the check to a qunit test now.